modin/experimental/engines/omnisci_on_ray/test/test_dataframe.py

```diff
@@ -56,7 +57,7 @@ def run_and_compare(
     force_lazy=True,
     force_arrow_execute=False,
     allow_subqueries=False,
-    **kwargs
+    **kwargs,
```

Comment:
This commit brings python 3.6+ syntax (formatted strings) to this file and so black formatter adds these trailing commas. [[1]](https://test-black.readthedocs.io/en/style-guide/style_guide/trailing_commas.html)